keras/wrappers/scikit_learn.py

```diff
@@ -316,7 +316,7 @@ def score(self, x, y, **kwargs):
 
         # Arguments
             x: array-like, shape `(n_samples, n_features)`
-                Test samples where n_samples in the number of samples
+                Test samples where n_samples is the number of samples
```

Comment:
You can use code markers around `n_samples` everywhere.

Reply:
Thanks for the tip, will do!